guides/source/action_text_overview.md

```diff
@@ -8,149 +8,141 @@ rich text content.
 
-Because these APIs were never fully specified or documented,
-and because WYSIWYG HTML editors are enormous in scope, each
-browser's implementation has its own set of bugs and quirks,
-and JavaScript developers are left to resolve the inconsistencies.
+Trix sidesteps these inconsistencies by treating`contenteditable` as an I/O device: when input makes its way to the editor, Trix converts that input into an editing operation on its internal document model, then re-renders that document back into the editor. This gives Trix complete control over what happens after every keystroke and avoids the need to use `execCommand` and the inconsistencies that come along with it.
 
-Trix sidesteps these inconsistencies by treating `contenteditable`
-as an I/O device: when input makes its way to the editor, Trix converts that input
-into an editing operation on its internal document model, then re-renders
-that document back into the editor. This gives Trix complete control over what
-happens after every keystroke, and avoids the need to use `execCommand` at all.
+**Action Text and the Trix Editor**
+
+The rich text content generated by the Trix editor is saved in its own RichText model that can be associated with any existing Active Record model in the application. In addition, any embedded images (or other attachments) can be automatically stored using Active Storage and associated with that RichText model.
+Active Storage is included as a dependency of Action Text. When it's time to render content, Action Text processes the content by sanitizing it first so that it's safe to embed directly into the page's HTML.
 
 ## Installation
 
-Run `bin/rails action_text:install` to add the Yarn package and copy over the necessary migration. Also, you need to set up Active Storage for embedded images and other attachments. Please refer to the [Active Storage Overview](active_storage_overview.html) guide.
+In order to install Action Text and start working with rich text content in your Rails app:
+
+1. Run `bin/rails action_text:install`
 
-NOTE: Action Text uses polymorphic relationships with the `action_text_rich_texts` table so that it can be shared with all models that have rich text attributes. If your models with Action Text content use UUID values for identifiers, all models that use Action Text attributes will need to use UUID values for their unique identifiers. The generated migration for Action Text will also need to be updated to specify `type: :uuid` for the `:record` `references` line.
+2. After the install command is complete, run `bin/rails db:migrate` to add the action_text and active_storage tables to your app
 
-After the installation is complete, a Rails app should have the following changes:
+The `action_text:install` command will do the following:
 
-1. Both `trix` and `@rails/actiontext` should be required in your JavaScript entrypoint.
+- Installs the Yarn packages for `trix` and `@rails/actiontext` and adds them to the `application.js`.
```

Comment:
I really like this: showing what changes should have taken place after running a command.

Reply:
I feel like this list should be right after the first step though.